village/src/components/SmurfForm.js

```diff
@@ -15,19 +19,35 @@ class SmurfForm extends Component {
     // add code to create the smurf using the api
 
     this.setState({
-      name: '',
-      age: '',
-      height: ''
+      name: event.target.name.value,
+      age: event.target.age.value,
+      height: event.target.height.value
     });
+
+    axios.post('http://localhost:3333/smurfs', this.state )
+    .then(res => {
+      console.log(res.data);
+      this.setState( {
+        newSmurf : res.data[res.data.length-1]
```

Comment:
Note that newSmurf is not actually going anywhere - it may be a good idea to lift this method up to the App component so that we can reset our state there and not require a refresh of the page